contracts/ERC20TimedMint.sol

```diff
@@ -1,23 +1,53 @@
 //SPDX-License-Identifier: Unlicense
-pragma solidity 0.8.6;
 
-import "./@openzeppelin/contracts/token/ERC20/ERC20.sol";
+pragma solidity 0.8.6;
 
+import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
 /**
  * @title ERC20TimedMint
- * @author Javier Gonzalez
+ * @author Javier Gonzalez and Marco Huberts
  * @dev Implementation of minting functionality for a mintable token.
  * @notice ERC20TimedMint inherits the ERC20 functionality and prevents 
- *         minting within a timeframe.
- */
+  *         minting within a timeframe.
+  */
 contract ERC20TimedMint is ERC20 {
-    uint256 timeUntilNexMint
 
-    constructor(
+    uint256 public timeUntilNextMint;
+    uint256 private mintCap;
```

Comment:
I think that we should also have a 'setter' for this function (only needs to be called internally) that way we can set to 0 if there will be no new tokens issued.